lib/common/logging.c

```diff
@@ -47,7 +47,7 @@ unsigned int crm_trace_nonlog = 0;
 bool pcmk__is_daemon = false;
 
 static unsigned int crm_log_priority = LOG_NOTICE;
-static GLogFunc glib_log_default = NULL;
+static guint pcmk__log_id = 0;
```

Comment:
Are you sure `g_log_set_handler()` can't return 0 as a valid ID? Hopefully glib's own default handler is 0

Reply:
I don't think it actually uses IDs for the default handler - at least, not on main.  Looking at the source, it maintains a separate default_log_handler variable that is just assigned a function, not a handler object that would have an ID.